core/src/migrations/000063-addSyncModeToDestination.ts

```diff
@@ -8,12 +8,12 @@ export default {
 
       // Pipedrive
       await migration.sequelize.query(
-        `UPDATE "destinations" SET "syncMode"='sync' WHERE "state"='ready' AND "type"='pipedrive-export'`
+        `UPDATE "destinations" SET "syncMode"='sync' WHERE "state"='ready' AND "type"='pipedrive-export' AND "locked" IS NULL`
```

Comment:
As it stands now, this does not do anything for code config. Since the syncMode is now required for the destination, when code config is validated (`grouparoo validate`) or when the server starts up, you will get an error saying "Sync mode is required for destination". The user then needs to update their code config accordingly.

Do we need to do anything else for code config?

Reply:
Let's discuss this one a little more.  Here are some thoughts about the migration that I want to understand:

* Does this mean we'll need a migration for every plugin as they gain SyncModes? If so, that doesn't seem great...
* Should the default value be `skip`? 
* Can you do something fancy with defaults for each type of plugin based on what the old behavior was closest too?  We could add the syncMode column now for all plugins even though it won't be used yet.  
* Or, should we call this a breaking change (and move to v0.3) and default all the existing destinations to the least dangerous option (`additive`), with a note for our users to check if that's what they want?